File: rgb.py
"""
Created by: Nina Mathew
Created on: March 21 2023
This program uses the nested loop statement
"""


def main() -> None:


    # process
    for r_value in range(0, 255):
        for g_value in range(0, 255):
            for b_value in range(0, 255):
                print("RGB({0}, {1}, {2})".format(r_value, g_value, b_value))
    print("\nDone.")


if __name__ == "__main__":
    main()

File: RGB.c
// Copyright (c) 2022 Nina Mathew All rights reserved
//
// Created by: Nina Mathew
// Created on: Mar 29, 2023
// This program guesses a number

#include <stdio.h>


int main() {
    // This functions uses RGB values

int r_Value = 0;
int g_Value = 0;
int b_Value = 0;

    for (int r_Value = 0; r_Value < 256; r_Value++) {
        for (int g_Value = 0; g_Value < 256; g_Value++) {
            for (int b_Value = 0; b_Value < 256; b_Value++) {
                printf("RGB(%d,%d,%d)\n", r_Value, g_Value, b_Value);
            }
        }
    }

    printf("\nDone.\n");
    return 0;
    }
